apps/charterafrica/src/pages/knowledge/[slug].page.js

```diff
@@ -0,0 +1,102 @@
+import Explainers from "@/charterafrica/components/Explainers";
+import PageInfo from "@/charterafrica/components/PageInfo";
+import ResearchComponent from "@/charterafrica/components/Research";
+import { payload } from "@/charterafrica/lib";
+import getGlobalProps from "@/charterafrica/utils/getGlobalProps";
+
+function Explainer({ blocks }) {
+  return blocks?.map((block) => {
+    switch (block?.slug) {
+      case "page-info":
+        return <PageInfo {...block} key={block.slug} />;
+      case "explainers":
+        return <Explainers {...block} key={block.slug} />;
+      case "research":
+        return <ResearchComponent {...block} key={block.slug} />;
+      default:
+        return null;
+    }
+  });
+}
+
+export async function getServerSideProps({
+  defaultLocale,
+  locale,
+  locales,
+  params: { slug },
+}) {
+  const lowerCasedSlug = slug?.toLowerCase();
+  const explainer = await payload.findPage(lowerCasedSlug, {
```

Comment:
```suggestion
  const knowledge = await payload.findPage(lowerCasedSlug, {
```
This is for all pages under knowledge... can't call it explainer